okd/src/README.md

```diff
@@ -0,0 +1,31 @@
+## Build and Run Microshift upstream without subscription/pull-secret
+
+- building the container from src
+  > `cd microshift-okd/src && ./build.sh` 
+  - this script will:
+    1. replace microshift assets images to OKD  upstream images
+    1. will build microshift RPMs and repo based on current sources.
+    1. will build micrsoshift_okd bootc container based on `centos-bootc:stream9`
+    1. apply upstream customization  (see below)
+
+- building the container with podman multistage build :
+  > `git clone https://github.com/openshift/microshift.git ~/microshift`
+  > `cd ~/microshift && podman build --no-cache --volume $(pwd)/:/src:Z -f okd/src/microshift-okd-multi-build.Containerfile`
```

Comment:
i had to add the  `--no-cache`  because microshift rpms are the artifacts of the `builder` stage and if we re-run the build it will not produce new RPMs (it will assume there is no change),.

Reply:
How does the `microshift-okd-multi-build.Containerfile` gets the `/src`?

Reply:
the current source  mounted  throughout  `--volume`

Reply:
That's why you need `--no-cache` - if there was `ADD/COPY . .` for source, then changing the files would invalidate the cache. Have you tried copying the sources to the container? I think it would be more aligned with how DockerContainer files are expected to be

Reply:
originally i've started with COPY, but the problem if your DockerFile inside sub-dir  then :+1: - you must  must run the build in the context of MICROSHIFT root.
 IE:
`cd ~/microshift && podman build okd/src/microshift-okd-multi-build.Containerfile .`

the problem is that microshift root ,  can become realy big , i can add `.dockerignore` to ignore few dirs ,
ie:
```
_output
.git
.github
```

Reply:
@pmtk , see the updated version

Reply:
It looks better.
We could also specify want we want to copy, although it's bit lengthy... On the other hand, we would be sure that there's no need to remember specific podman build invocation
```
COPY assets cmd etcd packaging pkg scripts vendor go.mod go.sum Makefile Makefile.kube_git.var Makefile.version.aarch64.var Makefile.version.x86_64.var /src
```